prism/src/parser/IdentUsage.java

```diff
@@ -125,11 +125,23 @@ public void checkIdent(String ident, ASTElement decl, String use) throws PrismLa
 		}
 	}
 
+	public IdentUsage deepCopyFields()
+	{
```

Comment:
Maybe the identDecls should also be deepCopied in a way that will preserve the structure.